### PR TITLE
update list and typography

### PR DESCRIPTION
**Pull request by Priya-Patel31**
added list and typography component

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
You are doing it great, you can merge.

**Comment by Manju-sree** [comment]
really looks clean css as name suggest. good to merge

File: component/lists/lists.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
+.outer-list-ctn{
+    display: flex;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    /* border:1px solid red; */
```
Comment (t007rushi) [inline component/lists/lists.css:5]: Remove this unwanted comments

Comment (Priya-Patel31) [reply component/lists/lists.css:5]: made changes!!

**Review by t007rushi — APPROVED** [review]

**Pull request merged**